07-routing-gossip.md

```diff
@@ -359,6 +381,12 @@ to be ordered in ascending order, unknown ones can be safely ignored.
 Additional fields beyond `addresses` may also be added in the future—with
 optional padding within `addresses`, if they require certain alignment.
 
+If a node signals `option_will_fund`, they are signaling that they
+will provide funding to a node at the stated terms. They also commit
+to a feerate they will charge for transmitting funds over the channel
+for the duration of the funding lease.
```

Comment:
The additional amount they will charge for onchain fees is only known after they reply to an open_channel2 request.